mobile/src/main.jsx

```diff
@@ -59,14 +52,14 @@ const renderAppWithPersistedState = persistedState => {
       isClientRegistered(client).then(clientIsRegistered => {
         if (clientIsRegistered) {
           const options = {
-            onError: onError(store, callback),
+            onError: onError(store.dispatch, store.getState, callback),
             onComplete: refreshFolder(store.dispatch, store.getState)
           }
           cozy.client.offline.startRepeatedReplication('io.cozy.files', 15, options)
           initBar()
           callback()
         } else {
-          onError(store, callback)()
+          onError(store.dispatch, store.getState, callback)()
```

Comment:
We must extract `callback` from the parameter and call it directly here (cf. https://github.com/cozy/cozy-files-v3/pull/179/files#r106179418)